packages/contracts-bedrock/src/L2/L1BlockIsthmus.sol

```diff
@@ -19,9 +20,9 @@ enum ConfigType {
 
 /// @custom:proxied
 /// @custom:predeploy 0x4200000000000000000000000000000000000015
-/// @title L1BlockInterop
-/// @notice Interop extenstions of L1Block.
-contract L1BlockInterop is L1Block {
+/// @title L1BlockIsthmus
```

Comment:
The smart contract diff looks good to me, It could be useful to submit the contracts as their own PR to make the review smaller